instance/models/instance.py

```diff
@@ -426,10 +426,14 @@ def provision(self):
         server = self.server_set.create()
         server.start()
 
+        def can_proceed():
```

Comment:
@itsjeyd nit: `can_proceed` is a bit too generic: maybe `server_accepts_ssh_commands`?